data/json/items/containers.json

```diff
@@ -483,7 +483,7 @@
         "bashing" : 6,
         "to_hit" : -4,
         "contains" : 240,
-        "material" : ["steel"],
+        "material" : "steel",
```

Comment:
Was noticed when referencing.
Items with a single material type did not use brackets, but those with multiple did.